### PR TITLE
Added translated template to snippet list

### PR DESCRIPTION
**Pull request by wachterjohannes**
__tasks:__

- [ ] gather feedback for my changes

__informations:__

| q                | a
| ---------------- | ---
| Fixed tickets    | none
| BC breaks        | none
| Documentation PR | none

### CODE REVIEW

**Comment by danrot** [comment]
I would add a test to that :smiley:

**Comment by wachterjohannes** [comment]
@danrot added a testcase

File: src/Sulu/Bundle/SnippetBundle/Tests/Functional/Controller/SnippetControllerTest.php [hunk 1]
```diff
@@ -76,6 +77,26 @@ public function testGetMany()
...
+                '/snippets?ids=%s&language=fr',
+                $this->hotel1->getUuid()
+            )
+        );
+        $response = $this->client->getResponse();
+
+        $this->assertEquals(200, $response->getStatusCode());
+        $res = json_decode($response->getContent(), true);
+
+        $res = reset($res['_embedded']['snippets']);
```
Comment (danrot) [inline src/Sulu/Bundle/SnippetBundle/Tests/Functional/Controller/SnippetControllerTest.php]: Do not abbreviate. Is that also done in the other test cases? Can you please change that everywhere?

**Comment by wachterjohannes** [comment]
@danrot fixed

**Pull request merged**